### PR TITLE
Add streaming Bible from Bible Brain API

### PR DESCRIPTION
**Pull request by mattleff**
Fixes https://trello.com/c/t6Fpf7Wq/16-bible-detail.

localhost_3000_en_bibles

### CODE REVIEW

**Review by narthur — APPROVED** [review]
Great work!

File: src/containers/bible/book.tsx [hunk 1]
```diff
@@ -1,188 +1,215 @@
...
+		),
+	});
 
 	return (
 		<Tease>
 			<Link href={makeBibleVersionRoute(languageRoute, id)}>
 				<a className={styles.hat}>
-					<BibleVersionTypeLockup unpadded />
-					<h4>{title}</h4>
+					<BibleVersionTypeLockup unpadded={true} label="KJV Bible" />
```
[the remark this answers is not in the document]
Comment (mattleff) [reply src/containers/bible/book.tsx:148]: Bibles are KJV English-only right now, so I decided we didn't need to localize things until that changes.

File: src/containers/bible/book.tsx [hunk 2]
```diff
@@ -1,188 +1,215 @@
...
+		const scroller = scrollRef.current;
+		scroller.scrollTo({ top: currentRef.current.offsetTop - 32 });
+		const saveScrollPosition = (e: Event) => {
+			setScrollPosition((e.target as HTMLElement).scrollTop);
+		};
+		scroller.addEventListener('scroll', saveScrollPosition);
+		return () => scroller.removeEventListener('scroll', saveScrollPosition);
+	}, [currentChapterNumber]);
 
 	// TODO: Remove this transformation when the API returns Recording type
```
[the remark this answers is not in the document]
Comment (mattleff) [reply src/containers/bible/book.tsx:69]: No, we will have Recording-type Bibles in the future, but for now we still need to transform them for the things expecting a Recording-like type.

File: src/containers/bible/book.tsx [hunk 3]
```diff
@@ -1,188 +1,215 @@
...
+			return;
+		}
+		const scroller = scrollRef.current;
+		scroller.scrollTo({ top: currentRef.current.offsetTop - 32 });
+		const saveScrollPosition = (e: Event) => {
+			setScrollPosition((e.target as HTMLElement).scrollTop);
+		};
+		scroller.addEventListener('scroll', saveScrollPosition);
+		return () => scroller.removeEventListener('scroll', saveScrollPosition);
+	}, [currentChapterNumber]);
```
[the remark this answers is not in the document]
Comment (mattleff) [reply src/containers/bible/book.tsx:67]: My understanding of exhaustive deps check is that it doesn't prevent you from adding to the dependencies specified (as in this case), it only prevents you from not having dependencies specified that you should have.

**Pull request merged**